go.mod

```diff
@@ -116,5 +130,12 @@ require (
 	golang.org/x/time v0.0.0-20181108054448-85acf8d2951c // indirect
 	google.golang.org/api v0.0.0-20181212003324-40e757e92c52
 	google.golang.org/genproto v0.0.0-20181109154231-b5d43981345b
+	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/vmihailenco/msgpack.v2 v2.9.1 // indirect
+	k8s.io/api v0.0.0-20190116205037-c89978d5f86d // indirect
+	k8s.io/apimachinery v0.0.0-20190116203031-d49e237a2683
+	k8s.io/client-go v7.0.0+incompatible
+	k8s.io/kubectl v0.0.0-20190502165022-ce8d9f55c93c
+	labix.org/v2/mgo v0.0.0-20140701140051-000000000287 // indirect
```

Comment:
Why we need mongo driver?

Reply:
Apperntally both `launchpad.net/gocheck` and `labix.org/v2/mgo` are added by mistake. I removed them.